### ISSUE TITLE
UI placement

### ISSUE THREAD

**Issue by kdzwinel** [arthurvr/timeline-url#18]
I find current placement of the UI (in the Sources, next to the 'watch' pane) awkward. It would be much better to introduce a new panel in the main menu (especially when upload feature is introduced).

**Comment by arthurvr** [comment]
The best place would be to have it in the `Timeline` tab, but that's not possible :cry:

**Comment by kdzwinel** [comment]
Yeah, that would be perfect - lets open a feature request for that on crbug ;)
